src/components/app/list-new/HelmAppList.tsx

```diff
@@ -88,28 +86,32 @@ export default function HelmAppList({
 
     useEffect(() => {
         if (serverMode == SERVER_MODE.EA_ONLY) {
-            setDataStateType(AppListViewType.LIST);
+            setDataStateType(AppListViewType.LIST)
```

Comment:
Won't we be showing syncing in case of EA_ONLY?

Reply:
Added